doc/src/vpr/index.rst

```diff
@@ -32,3 +32,5 @@ The purpose of VPR is to make the packing, placement, and routing stages of the
 
    file_formats
    debug_aids
+
+   fasm
```

Comment:
It seems like the fasm documentation should go under a new 'utils' section instead of under VPR.